client-src/elements/chromedash-guide-stage-page.ts

```diff
@@ -264,13 +265,16 @@ export class ChromedashGuideStagePage extends LitElement {
         stageId,
       });
 
+      const disabledHelpText = getDisabledHelpText(field, feStage);
       return html`
         <chromedash-form-field
           name=${field}
           index=${index}
           value=${value}
           .fieldValues=${this.fieldValues}
           .feature=${formattedFeature}
+          helpText="${disabledHelpText}"
+          ?disabled=${disabledHelpText}
```

Comment:
The code that you have is fine, but one idea to make this more compact is to use one attribute instead of two, like `disabledReason=${getDisabledHelpText(...)}` and then have that be handled inside chromedash-form-field to do both the disabling and the help text.

Reply:
This is a great suggestion, and I've updated the implementation with this approach 🙂